src/components/rangeslider/draw.js

```diff
@@ -92,8 +95,13 @@ module.exports = function(gd) {
         opts._height = (fullLayout.height - margin.b - margin.t) * opts.thickness;
         opts._offsetShift = Math.floor(opts.borderwidth / 2);
 
-        var x = margin.l + (graphSize.w * domain[0]),
-            y = fullLayout.height - opts._height - margin.b;
+        var x = margin.l + (graphSize.w * domain[0]);
+
+        var y = (
+            margin.t + graphSize.h * (1 - oppDomain[0]) +
+            tickHeight +
+            opts._offsetShift + constants.extraPad
+        );
```

Comment:
https://github.com/plotly/plotly.js/commit/5a649f03339c155893eeec882b0bd15ffe0e5968 gives

![image](https://cloud.githubusercontent.com/assets/6675409/23242732/919b4a7c-f948-11e6-80d9-c323d057a7fc.png)

is that what you had in mind?